File: study_view/src/main/java/com/view/recycleView/RcvActivity.java
package com.view.recycleView;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.sn.study_pic.R;


public class RcvActivity extends AppCompatActivity {

    private RecyclerView mRecyclerView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_rcv);
        mRecyclerView = (RecyclerView) findViewById(R.id.rcy_list);
        mRecyclerView.setLayoutManager(new LinearLayoutManager(this));
    }

}

File: study_view/src/main/java/com/view/recycleView/RcvActivity.kt
package com.view.recycleView

import android.os.Bundle
import android.view.View
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.sn.study_pic.R
import com.view.BaseDataActivity
import com.view.StickyScrollView.ListAdapter

class RcvActivity : BaseDataActivity() {
    private var mRecyclerView: RecyclerView? = null

    override fun initView(inflate: View) {
        val listAdapter = ListAdapter(baseContext)
        mRecyclerView = findViewById<View>(R.id.rcy_list) as RecyclerView
        mRecyclerView?.layoutManager = LinearLayoutManager(this)
        mRecyclerView?.adapter = listAdapter
        listAdapter.setData(arrayListData)
    }

    override fun setLayout(): Int {
        return R.layout.activity_rcv;
    }
}